cluster/v1beta1/0000_03_clusters.open-cluster-management.io_placementdecisions.crd.yaml

```diff
@@ -15,14 +15,14 @@ spec:
   - name: v1beta1
     schema:
       openAPIV3Schema:
-        description: "PlacementDecision indicates a decision from a placement PlacementDecision
-          should has a label cluster.open-cluster-management.io/placement={placement
+        description: "PlacementDecision indicates a decision from a placement. PlacementDecision
```

Comment:
Suggestion: "PlacementDecision indicates a decision from a placement. PlacementDecision
          must have a `cluster.open-cluster-management.io/placement={placement
          name}`  label to reference a certain placement. \n If a placement has spec.numberOfClusters
          specified, the total number of decisions contained in the status.decisions
          of PlacementDecisions must be the same as NumberOfClusters. Otherwise, the
          total number of decisions must equal the number of ManagedClusters that
          match the placement requirements. \n Some of the decisions might be empty
          when there are not enough ManagedClusters to meet the placement requirements."